vita3k/modules/SceLibKernel/SceLibKernel.cpp

```diff
@@ -97,40 +97,40 @@ EXPORT(int, sceClibMemchr) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceClibMemcmp, const void *ptr1, const void *ptr2, SceSize num) {
-    return memcmp(ptr1, ptr2, num);
+EXPORT(int, sceClibMemcmp, const void *s1, const void *s2, SceSize len) {
+    return memcmp(s1, s2, len);
```

Comment:
Why 🤔

Reply:
Because explain to Fran is good name ^^ like all orher

Reply:
ok